### PR TITLE
Prettifying long commit histories with Git

### PR DESCRIPTION
**Pull request by sdvassil**
screencapture-127-0-0-1-4000-2021-03-17-17_18_35

### CODE REVIEW

**Comment by sdvassil** [comment]
@tgharold When you get a chance, can you please take a look at this to make sure I'm not off my rocker?

File: _posts/2021-03-17-prettifying-long-commit-histories.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+However, numerous options can make the **git log** command more robust. These helpers can narrow down, embellish, or prettify results.
+
+Here we focus on the **oneline** option, which serves as a shorthand for:
+
+`--pretty=online --abbrev-commit`
+
+## Time for some Git magic
+
+With the steps below, you can generate a complete list of PRs in your CLI. The assumption is that the upstream **master** branch is the source of truth for your project and the development branch.
```
Comment (tgharold) [inline _posts/2021-03-17-prettifying-long-commit-histories.md:37]: first use of the "CLI" acronym? or the "PR" acronym?

File: _posts/2021-03-17-prettifying-long-commit-histories.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    ![GitKraken Sample Commit Graph](/images/documentation/gitkraken-commit-graph-git-log-oneline-example.png)
+
+4. Find the older hash. In the real world, this could be the latest commit deployed in production. In our example, the value **2f380ba** is the last commit for the 2021-03-08 production branch.
+5. Similarly, get the more recent hash, which theoretically represents the tip of your release branch. In our example, the value **14fbcce** is the last commit for the 2021-03-10 release branch.
+
+6. After gathering all details, run the command below in the **master** branch of your local repository:
+
+    `git log --oneline 2f380ba..14fbcce | grep 'Merge pull'`
+
+7. Pull requests between the two hashes display per line. The **grep** option lets you search keywords in commit messages. Therefore, you're able to isolate just the PRs, without worrying about the 250 commit limit.
```
Comment (tgharold) [inline _posts/2021-03-17-prettifying-long-commit-histories.md:52]: >Pull requests between the two hashes display per line.
weird wording? "display one per line"?

File: _posts/2021-03-17-prettifying-long-commit-histories.md (hunk 1 above)
Comment (sdvassil) [reply _posts/2021-03-17-prettifying-long-commit-histories.md:37]: Good point on CLI. Made an assumption. Will update. The first sentence in the post addresses the PR acronym.

**Review by tgharold — APPROVED** [review]

**Comment by hougasian** [comment]
nicely done @sdvassil !

**Pull request merged**